sale_exception/views/sale_view.xml

```diff
@@ -95,6 +96,7 @@
             <field name="name">sale_exception.view_order_tree</field>
             <field name="model">sale.order</field>
             <field name="inherit_id" ref="sale.view_order_tree"/>
+            <field name="groups_id" eval="[(4, ref('base.group_user'))]"/>
             <field name="arch" type="xml">
```

Comment:
It makes more sense to me to assign this to the sale user

Reply:
Change in this commit https://github.com/OCA/sale-workflow/pull/394/commits/aa1a7eb321e869448c3008326ecadfbf41149e12

Reply:
Well, this commit is not here in this PR...